### ISSUE TITLE
Separating the CI for docs

### ISSUE THREAD

**Issue by LecrisUT** [scikit-build/scikit-build-core#381]
Since there are CIs for doc examples and the whole stack of python/cmake/os can take some time to build, how about requiring only the docs CI when `/docs` and related included files are changed. Packit does not support this yet though, but I'll open an issue for it.

**Comment by henryiii** [comment]
Yes, I’ve done this recently for https://github.com/scientific-python/cookie (based on https://github.com/PyPA/build)

**Comment by LecrisUT** [comment]
Oh wow, that's quite a neat implementation, thanks for the reference. I will be borrowing that design
